src/main/java/Main.java

```diff
@@ -0,0 +1,8 @@
+import java.io.IOException;
+
+public class Main{
+    public static void main(String[] args) throws DukeException, IOException {
+        Duke LamboInstance = new Duke();
```

Comment:
Variable names should be named appropriately with the first letter in lowercase.
```suggestion
        Duke lamboInstance = new Duke();
```